simpleTCPServer.py

```diff
@@ -48,7 +48,7 @@ def start(self):
 
                         for frame in self.recording:
                             semicolonByte = bytes(";", 'utf-8')
-                            connection.sendall(frame + semicolonByte)
+                            connection.sendall(framwife + semicolonByte)
```

Comment:
Typo? Because I don't see framwife used anywhere else in this file

Reply:
I think the cat did that